index.html

```diff
@@ -44,8 +48,169 @@ <h1 class="prmrytxt1">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div >
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>
+    <div class="port1">
+      <img src="images/card2pic.png" alt="Tonic">
+      <div>
+        <h2 class="title">Multi-Post <br> Stories</h2>
+        <div>
+          <span class="clnt">CANOPY</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">Back End Dev</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">2015</span>
+        </div>
+      </div>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>
+    <div class="port1">
+      <img src="images/card3pic.png" alt="Tonic">
+      <div>
+        <h2 class="title">Tonic</h2>
+        <div>
+          <span class="clnt">CANOPY</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">Back End Dev</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">2015</span>
+        </div>
+      </div>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>    
+    <div class="port1">
+      <img src="images/card4pic.png" alt="Tonic">
+      <div>
+        <h2 class="title">Multi-Post <br> Stories</h2>
+        <div>
+          <span class="clnt">CANOPY</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">Back End Dev</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">2015</span>
+        </div>
+      </div>
+      <p >
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
```

Comment:
[OPTIONAL] I think Your button states have not been fully implemented as given by the design, I suggest you kindly make the necessary changes as I think it is more professional to always stick to the given design🤗
kindly take note of the different states in the image below.
<img width="285" alt="saveButnState" src="https://user-images.githubusercontent.com/64643995/212461754-92fcd893-3205-46ce-97a6-e8cd0b7723a6.png">